src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/CognitiveServices.ContentModerator.Tests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  
+  <PropertyGroup>
+    <PackageId>ContentModerator.Tests</PackageId>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.ContentModerator.Tests</AssemblyName>
+    <Description>ContentModerator.Tests Class Library</Description>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="TestDataSources\NormalImages*.*">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
@v-sodsou please remove line 23 to 25

Reply:
@shahabhijeet  System.Net.Http is being used in test project.It gives me a build error if I remove it.